active/0002-centalized-mgmt-resource-types-provider-versions.md

```diff
@@ -70,64 +70,51 @@ provider az
 provider kubernetes as k8s // uses optional aliasing of the
 ```
 
-The resolution of the identifiers will be determined from inspection of the closest `bicepconfig.json` to the file merged with the builtin configuration. We propose to add a new section `providers` and `implicitProviders`.
+The resolution of the identifiers will be determined from the closest `bicepconfig.json` to the file merged with the default configuration. We propose the following changes:
+
+- Add new `providers` and `implicitProviders` sections to the `bicepconfig.json`.
+- Use the scheme `builtin:` as a sentinel value to declare the use of the NuGet package reference for a resource types provider
+- Use the scheme `br:` to prefix the fully qualified address of a provider package
 
 Example: The new default configuration
 ```json
 {
   // prior bicepconfig.json sections
   // ...
   "providers": {
-    "az": {
-      "builtIn": true
-    },
-    "kubernetes":{
-      "builtIn": true
-    },
-    "microsoftGraph":{
-      "builtIn": true
-    }
+    "az": "builtin:",
```

Comment:
I might missed something in the discussion - why is an extra `:` needed at the end?

Reply:
We need to disambiguate a local path from the sentinel value, @jeskew  offered that if we use `builtin:`, this value is a special scheme prefix ( other scheme prefixes are `br:`, `ts:`), this way we don't need to special case a scenario where a user wants to fetch from a folder with the name **builtin**